### PR TITLE
feat(test wallet): Adds testWallet function to WalletDAO PE-1809

### PR DESCRIPTION
**Pull request by arielmelendez**

### CODE REVIEW

File: src/wallet_dao.ts [hunk 1]
```diff
@@ -144,4 +144,21 @@ export class WalletDAO {
 			throw new Error(`Transaction failed. Response: ${response}`);
 		}
 	}
+
+	// Attempts to sign, without submitting to Arweave, a stubbed out transaction with the provided wallet
+	async testWallet(wallet: Wallet): Promise<void> {
```
Comment (fedellen) [inline src/wallet_dao.ts:149]: I'm not sure that `testWallet` is a clear enough name here. When I read the method I first assume that I will be using this to create a test wallet for development purposes

Consider `isValidWallet` or `debugWallet` or similar

Comment (fedellen) [inline src/wallet_dao.ts:149]:
```suggestion
	public async testWallet(wallet: Wallet): Promise<void> {
```

**Review by fedellen — COMMENTED** [review]
Looks good! We should absolutely include this functionality as part of our wallet utils

Left a comment on the method name, everything else is peachy 🍑

**Review by matibat — COMMENTED** [review]
The CORE-side is looking good. I'm just wondering why the CLI-side PR is closed

**Comment by matibat** [comment]
We are gonna hang on this for now. It might be addressed in the future, but not for now.

**Pull request closed without merging**